Source/ACE/Network/GameAction/GameActionOpcode.cs

```diff
@@ -3,13 +3,16 @@
     // order by opcode name
```

Comment:
I clearly missed that!  I've made another commit and fixed the ordering of opcodes in both GameEventOpcode and GameActionOpCode.